stream_alert/rule_processor/threat_intel.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import csv
+import gzip
+import os
+
+from stream_alert.rule_processor import LOGGER
+
+class ThreatIntel(object):
+    """Load intelligence from csv gz files into a dictionary."""
+
+    def __init__(self, intel_dir, delimiter=','):
+        """Initialize class instance variables
+        Args:
+            intel_dir (str): Location where stores compressed intelligence files
+
+        Instance variables:
+            self._intel_dict (dict): A dictionary to store all intelligence read
+                from *.csv.gz files
+            self._intel_dir (str):  Location where stores compressed intelligence
```

Comment:
`self._intel_dir (str):  Directory path that contains the threat intelligence files`